.github/workflows/canary-integration-test.yml

```diff
@@ -1,5 +1,10 @@
 name: Canary integration tests
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+    - '**.go'
```

Comment:
There may be some PRs that we want to run the full tests, even though these files didn't change, for example if we update the base operator image. Could we also trigger these actions with a label when we want to override the file extensions?

Reply:
I don't think it's possible :-/

Reply:
Not possible...

Reply:
Ok if we can't control it with a label, then what if we add more paths to trigger the tests, such as `build/*`, `tests/*`, and `images/*`? I'd rather run the tests too often than miss sometimes when needed.

Reply:
sounds good

Reply:
Build on image changes?

Reply:
changes under `images` are rare, but yes I think it makes sense to run integration tests in that case